internal/kinds/capp/controllers/capp_controller.go

```diff
@@ -13,17 +13,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	"github.com/dana-team/container-app-operator/internal/status"
+	"github.com/dana-team/container-app-operator/internal/kinds/capp/status"
 
-	"github.com/dana-team/container-app-operator/internal/finalizer"
+	"github.com/dana-team/container-app-operator/internal/kinds/capp/finalizer"
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/tools/record"
 	ctrl "sigs.k8s.io/controller-runtime"
 
 	cappv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
-	rmanagers "github.com/dana-team/container-app-operator/internal/resource-managers"
+	rmanagers "github.com/dana-team/container-app-operator/internal/kinds/capp/resource-managers"
```

Comment:
We need to rename the `resource-managers` package, since the directory name and package name are not the same